src/utils/formDataUtils.ts

```diff
@@ -111,7 +111,7 @@ const getPatronAgencyType = (agencyTypeParam?) => {
  * getLocationValue
  * Map the location value from the form field into the string value.
  */
-const getLocationValue = (location = "en", lang = "en"): string => {
+const getLocationValue = (location = "us", lang = "en"): string => {
   const defaultLocation = ipLocationMessageTranslations[lang]["us"];
```

Comment:
Might be still a little cloudy in my head. But now I am thinking that maybe `defaultLocation` should be set to ` const defaultLocation = ipLocationMessageTranslations["en]["us"];` since it should work as a fallback if something does not go well with the location/lang passed? But I might be wrong here..

Reply:
Yeah, and there's a simpler way to do this. Added in latest update.